lms/envs/common.py

```diff
@@ -41,6 +41,8 @@
 # The display name of the platform to be used in templates/emails/etc.
 PLATFORM_NAME = "Your Platform Name Here"
 CC_MERCHANT_NAME = PLATFORM_NAME
+# Shows up in the platform footer, eg "(c) COPYRIGHT_YEAR"
+COPYRIGHT_YEAR = "2015"
```

Comment:
I think a manual date is the best approach.  You only have to tweak it once a year, and this is centralized at least.